app/views/advanced/_numismatics_advanced_search_form.html.erb

```diff
@@ -0,0 +1,62 @@
+<%= form_tag search_catalog_path, :class => 'advanced form-horizontal', :method => :get do  %>
+  <div class="advanced row">
+    <% unless (search_context_str = render_search_to_s(advanced_search_context)).blank? %>
+      <div class="constraints well search_history">
+        <h4 class="advanced-search-context"><%= t 'blacklight_advanced_search.form.search_context' %></h4>
+        <%= search_context_str %>
+      </div>
+    <% end %>
+    <div class="col-md-5">
```

Comment:
can you change this to col-md-6

Reply:
Because of the offset, this looks better as column width 5. I changed the second column to 7 to make up the difference.